playbooks/common/openshift-cluster/redeploy-certificates/router.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Load lib_openshift modules
+  include_role:
```

Comment:
Should probably use `roles` on the play below rather than include_role here. I'm not even sure include_role is valid at this level.